importer/trickle/trickle_test.go

```diff
@@ -19,10 +19,23 @@ import (
 	u "gx/ipfs/QmWbjfz3u6HkAdPh34dgPchGbQjob6LXLhAeCGii2TX69n/go-ipfs-util"
 )
 
-func buildTestDag(ds merkledag.DAGService, spl chunk.Splitter) (*merkledag.ProtoNode, error) {
+type UseRawLeaves bool
+
+const (
+	ProtoBufLeaves UseRawLeaves = false
```

Comment:
Declaring a type for this seems like overkill.

Reply:
It makes things easier to read.

Reply:
Okay, yeah. I'm fine with it (after reading through the tests)